Wrappers/Python/cil/optimisation/functions/MixedL21Norm.py

```diff
@@ -72,15 +72,24 @@ def _proximal_step_numpy(tmp, tau):
     '''
     # Note: we divide x by tau so the cases of tau both scalar and 
     # DataContainers run
-    tmp /= np.abs(tau, dtype=np.float32)
-    res = tmp - 1
+    try:
+        tmp = np.abs(tau, dtype=np.float32)
+    except np.core._exceptions._UFuncInputCastingError:
+        tmp = tau.abs()
+    
+    
+    arr /= tmp
+    res = arr - 1
     res.maximum(0.0, out=res)
```

Comment:
Does this work if arr is a numpy array?

Reply:
Probably it would fail when we do fill. Well spotted.